### DISCUSSION TITLE
I used the official example, but the error was reported

### DISCUSSION THREAD

**Discussion by whyiyu** [vectordotdev/vector#9902] [Q&A] [answered]
# version
vector 0.15.0 (x86_64-pc-windows-msvc 994d812 2021-07-16)

# transforms
`
[transforms.del_other_field]
type = "remap"
inputs = ["inputs"]
source = """
# dosomething
.test = find!("foobar", r'b.r')
"""
`

# error
```
Nov 04 23:23:26.774 ERROR vector::topology: Configuration error. error=Transform "del_other_field":
[0m[1m[38;5;9merror[E105][0m[1m: call to undefined function[0m
  [0m[36m┌─[0m :2:9
  [0m[36m│[0m
[0m[36m2[0m [0m[36m│[0m .test = [0m[31mfind[0m!("foobar", r'b.r')
  [0m[36m│[0m         [0m[31m^^^^[0m
  [0m[36m│[0m         [0m[31m│[0m
  [0m[36m│[0m         [0m[31mundefined function[0m
  [0m[36m│[0m         [0m[36mdid you mean "int"?[0m
  [0m[36m│[0m
  [0m[36m=[0m learn more about error code 105 at https://errors.vrl.dev/105
  [0m[36m=[0m see language documentation at https://vrl.dev
```

**Comment by spencergilbert** [comment, marked as answer]
`find` was added in [0.17.0](https://vector.dev/releases/0.17.0/), you'll need to update your local version.

**Reply by whyiyu** [reply]
Thank you for your reply. In addition, I have 40000 pieces of data. I want to judge whether a field is in it. If it is, it will be filtered. What should I do in VRL?

**Reply by spencergilbert** [reply]
There are a number of "lookup" functions [here](https://vector.dev/docs/reference/vrl/functions/) depending on the type of data being used.

**Reply by whyiyu** [reply]
well,Thank you for your reply.
